merp_outgoing_routing/views/stock.xml

```diff
@@ -6,7 +6,7 @@
         <field name="model">stock.location</field>
         <field name="inherit_id" ref="stock.view_location_tree2" />
         <field name="arch" type="xml">
-            <xpath expr="//field[@name='complete_name']" position="after">
```

Comment:
And final comment - description should be reworked according to latest changes

Reply:
updated a little bit